opentracing-jaxrs2/src/main/java/io/opentracing/contrib/jaxrs2/server/ServerTracingFilter.java

```diff
@@ -64,6 +67,7 @@ public void filter(ContainerRequestContext requestContext) {
         if (tracer != null) {
 
             Tracer.SpanBuilder spanBuilder = tracer.buildSpan(operationNameProvider.operationName(requestContext))
+                    .ignoreActiveSpan()
```

Comment:
This could probably always have been in the code as far as I can tell; the thing that actually seems to be more influential is:
```
if (parentSpanContext != null) {
  spanBuilder.asChildOf(parentSpanContext);
}
```
below.